fisher.fish

```diff
@@ -117,7 +117,7 @@ function $fisher_cmd_name -d "fish plugin manager"
     command mkdir -p {"$fish_path","$fish_config"}/{conf.d,functions,completions} "$fisher_config" "$fisher_cache"
     or return 1
 
-    set -l completions "$fish_config/completions/$fisher_cmd_name.fish"
+    set -l completions "$fish_path/completions/$fisher_cmd_name.fish"
```

Comment:
With `$fish_path` set currently, all completions get installed to `$fish_path/completions/` anyway, this just makes `$fisher_cmd_name.fish` follow the same behaviour.